ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -213,6 +277,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-shared-vpc-phz-freeipa-f28
+  cron: 34 14 11,27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: idp-freeipa
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
 - as: aws-ipi-shared-vpc-phz-sts-openldap-fips-f28
```

Comment:
By a chance, I saw this job in a similar situation as https://github.com/openshift/release/pull/54732/files#r1690749592, cc @xingxingxia to confirm if need to run `idp-openldap` step.

Reply:
@jianlinliu , would like to cover openldap idp in upgrade, you can keep it here.

Reply:
If that, sounds like we also can move this e2e job to mult-arch ci to avoid duplicated copy for upgrade ci.

Reply:
@jianlinliu Got you. But is this moving very required? I searched: https://github.com/search?q=repo%3Aopenshift%2Frelease++%22ref%3A+idp-openldap%22&type=code . All e2e and upgrade are in amd ci. I'd like to keep them not moved : )

Reply:
If you can help take moving the e2e and upgrade ones to multi ci for long-term, that is fine : )

Reply:
The reason why they are existing in amd ci, because most of team member are more familiar with amd arch, so they were added to amd ci, but we already received several requirements:
1. save budget as much as possible, using arm is cheaper than amd.
2. mult-arch team ask the whole team to cover mult-arch payload as much as possible, if the feature does not rely on arches. 

We already spent a lot of time in migrating signle-arch ci jobs to mult-arch ci, so yeah, if I can move all of them to multi-arch ci unless some specific ci only supported in single-arch ci.

Reply:
@jianlinliu very much understood now, thanks for patient explaining. If this was not yet told to wider @ openshift-qe, it is very worthy to spread to @ openshift-qe : ) Also https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md can highlight your above points.

Reply:
Thanks @jianlinliu I will remove this one, and you can add the same job to multi-arch.

Reply:
@xingxingxia correct my words, because all the base profiles aws-ipi-shared-vpc-phz-* are existing in amd ci, so I wound not migrate it to multi-arch ci for this job. 

so @JianLi-RH I guess you can keep it in this PR.